src/components/feed/FeedTree/Node.tsx

```diff
@@ -191,7 +216,9 @@ const NodeWrapper = (props: NodeWrapperProps) => {
       scale = Math.log10(end.getTime() - start.getTime()) / 2
     }
   } else if (overlayScale === 'size') {
-    // props.data.item?.
+      if (outputSize) {
+        scale = outputSize / (1024 ** 2) / 10
```

Comment:
the size of nodes will probably need to be scaled relative to the maximum size in the feed.

Reply:
I refactored this to scale to the node window, with the scale maxing out at 700MB.  This seems to be able to represent a range of file sizes, although it's a bit hard to differentiate when the sizes are small (under 50MB or so).  The other option would be to set the max relative to the size of the whole feed, which is what I think you're suggesting, however, we would have to account for changes in the size as more nodes are added, which might be confusing.  Which do you think is better?